### PR TITLE
Added Card Component

### PR DESCRIPTION
**Pull request by prakashsakari**
Kindly review my card component, any feedback is appreciated. Thank you

### CODE REVIEW

File: Card/badge-card.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="author" content="Prakash Sakari">
+    <title>Vertical Card with Badge</title>
```
Comment (rohanmathur91) [inline Card/badge-card.html:9]:
```suggestion
You can also add a meta tag for the description.
```

**Review by rohanmathur91 — APPROVED** [review]
Looks good.

File: Card/horizontal-card.html [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    <meta name="author" content="Prakash Sakari">
+    <title>Horizontaol Card</title>
+    <link rel="stylesheet" href="/Card/card.css">
+</head>
+
+<body>
+    <div class="container">
+        <div class="card-horizontal c-flex">
+            <div class="card-hori-image-container card-badge">
+                <img class="card-image" src="/Card/assets/shoes.jpg" alt="">
```
Comment (Naveenchandar) [inline Card/horizontal-card.html]: Please provide alt description for accessibility

File: Card/badge-card.html [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+                </div>
+                <div class="cta-btn">
+                    <button class="button btn-primary btn-icon d-flex cursor btn-margin">
+                        <img src="/Card/assets/cart.png" alt="cart"> Add To Cart</button>
+                </div>
+                
+            </div>
+            
+        </div>
+
```
Comment (Naveenchandar) [inline Card/badge-card.html]: You can remove the extra spaces.

File: Card/card-dismiss.html [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+                </div>
+                <div class="cta-btn">
+                    <button class="button btn-primary btn-icon d-flex cursor btn-margin">
+                        <img src="/Card/assets/cart.png" alt="cart"> Add To Cart</button>
+                </div>
+                
+            </div>
+            
+        </div>
+
```
Comment (Naveenchandar) [inline Card/card-dismiss.html]: You can remove the extra spaces.

File: Card/card-shadow.html [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+                <div class="cta-btn">
+                    <button class="button btn-primary btn-icon d-flex cursor btn-margin">
+                        <img src="/Card/assets/cart.png" alt="cart"> Add To Cart</button>
+                </div>
+                
+            </div>
+            
+        </div>
+
+
```
Comment (Naveenchandar) [inline Card/card-shadow.html]: You can remove the extra spaces.

File: Card/vertical-card.html [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+                <div class="cta-btn">
+                    <button class="button btn-primary btn-icon d-flex cursor btn-margin">
+                        <img src="/Card/assets/cart.png" alt="cart"> Add To Cart</button>
+                </div>
+                
+            </div>
+            
+        </div>
+
+
```
Comment (Naveenchandar) [inline Card/vertical-card.html]: You can remove the extra spaces.

**Review by Naveenchandar — APPROVED** [review]
Looks good

File: Card/card-dismiss.html (hunk 4 above)
Comment (prakashsakari) [reply Card/card-dismiss.html]: ok will do that

File: Card/badge-card.html (hunk 3 above)
Comment (prakashsakari) [reply Card/badge-card.html]: Thanks, updated

**Pull request merged**